### ISSUE TITLE
AirWallex Payment Elements NPM Repo has no home...

### ISSUE THREAD

**Issue by dillonbailey** [airwallex/airwallex-payment-demo#74]
When navigating to GitHub from https://www.npmjs.com/package/airwallex-payment-elements - the link to https://github.com/airwallex/airwallex-payment-elements produces a 404. This makes it challenging to know where, if anywhere, we should be logging issues when using the library.

**Comment by chao-ding-awx** [comment]
Yeah, you're right about this. We migrate our repo from github to gitlab recently and I think we need to update the link as well. Thank you for reporting.
